bundles/net.tourbook.common/src/net/tourbook/common/util/AdvancedMenuForActions.java

```diff
@@ -75,10 +75,63 @@ public class AdvancedMenuForActions {
 
    private class ContextArmListener implements ArmListener {
 
+      private synchronized void onAnimation10Start() {
+
+         if (_isAnimating) {
+            return;
+         }
+
+         _isAnimating = true;
+
+         _consumedAnimationTime = 0;
+
+         _display.asyncExec(_animationRunnable);
+      }
+
+      private void onArmEvent(final ArmEvent event) {
+
+         final MenuItem menuItem = (MenuItem) event.widget;
+
+         if (_isAutoOpen && menuItem.isEnabled()) {
+
+            final Object itemData = menuItem.getData();
+            if (itemData instanceof final ActionContributionItem actionContributionItem) {
+
+               final String itemId = actionContributionItem.getId();
+               final String actionId = _actionContributionItem.getId();
```

Comment:
Can you revert this changed code.

I put as less code as possible into an anonymous class because when debugging and modifying it, you need an app restart

Reply:
Reverted